self-service-kiosk/src/app/store/preparation-store.js

```diff
@@ -1,40 +1,68 @@
 import {createAsyncThunk, createSlice} from "@reduxjs/toolkit";
 import catalogConfig from "../config/catalog-config.js";
-import {getMenusAsync} from "./catalog-store.js";
-
-const initialState = {
-    preparationOrders: [],
-    readyOrders: [],
-};
+import {preparationConfig} from "../config/preparation-config";
 
+// Fetch my current configuration
+const preparationsMethods = preparationConfig[process.env.REACT_APP_CONFIG];
 
```

Comment:
cet import pas forcement necessaire

Reply:
From my point of view, this makes the code easier to understand and we can easily change the configuration branching.